File: 3302/3302.c
// bee 3302 - Resposta Certa
// Alonso Martins
// 20/02/2024
#include <stdio.h>

int main()
{
    int N, resposta;

    scanf("%d", &N);

    for (int i = 0; i < N; i++)
    {
        scanf("%d", &resposta);
        printf("resposta %d: %d\n", i + 1, resposta);
    }
    return 0;
}

File: 3302/3302.java
// bee 3302 - Resposta Certa
// Alonso Martins
// 20/02/2024
import java.util.Scanner;

public class Main
{
	public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        int n = in.nextInt();
        
        for(int i = 0; i < n; i ++) {
            System.out.println("resposta "+(i+1)+": "+in.nextInt());
        }
	}
}
